File: libPrjPython/Media.py
class Media(object):
    
    def __init__(self, callno, title, subject, notes):
        self.callno = callno
        self.title = title
        self.subject = subject
        self.notes = notes

    def compare_callno(self, srch_callno):
        return srch_callno in self.callno
    
    def compare_title(self, srch_title):
        return srch_title in self.title

    def compare_subject(self, srch_subject):
        return srch_subject in self.subject

    def display(self):
        print "Call No.             : ", self.callno
        print "Title                : ", self.title
        print "Subject              : ", self.subject
       

File: librarySystem/Media.cpp
#include <iostream>
#include "Media.h"
#include "Book.h"

using namespace std;


Media::Media(string &cNo, string & titl, string &sub, string &note)
{
	this->callNo = cNo;
	this->title = titl;
	this->subjects = sub;
	this->notes = note;
}

Media::~Media()
{
}

bool Media::compare_callNumber(const string &callNum) const
{
	if(this->callNo.find(callNum) != std::string::npos)
	{
		return true;
	}
	else
		return false;
}

bool Media::compare_title(const string &title) const
{
	if(this->title.find(title) != std::string::npos)
	{
		return true;
	}
	else
		return false;
}

bool Media::compare_subject(const string &subject) const
{
	if(this->subjects.find(subject) != std::string::npos)
	{
		return true;
	}
	else
		return false;
}


void Media::display() const
{
	cout<<"Call Number------------"<<this->callNo<<endl;
	cout<<"Title------------------"<<this->title<<endl;
	cout<<"Subject----------------"<<this->subjects<<endl;
}
